react/features/invite/components/add-people-dialog/web/DialInNumber.tsx

```diff
@@ -1,16 +1,21 @@
-import React, { Component } from 'react';
-import { WithTranslation } from 'react-i18next';
+import React, { useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useDispatch } from 'react-redux';
 
-import { translate } from '../../../../base/i18n/functions';
 import Icon from '../../../../base/icons/components/Icon';
-import { IconCopy } from '../../../../base/icons/svg';
+import { IconCheck, IconCopy } from '../../../../base/icons/svg';
+import Tooltip from '../../../../base/tooltip/components/Tooltip';
 import { copyText } from '../../../../base/util/copyText.web';
+import { showSuccessNotification } from '../../../../notifications/actions';
+import { NOTIFICATION_TIMEOUT_TYPE } from '../../../../notifications/constants';
 import { _formatConferenceIDPin } from '../../../_utils';
 
+let mounted: boolean;
+
 /**
  * The type of the React {@code Component} props of {@link DialInNumber}.
  */
-interface IProps extends WithTranslation {
+interface IProps {
```

Comment:
WithTranslation exposes the translate function `t`, it adds it to props, so then you don't need to do `const { t } = useTranslation();`, instead get t directly from props, `function DialInNumber({ conferenceID, phoneNumber, t }: IProps) {`